platform/security/sts/security-sts-attributequerycommon/src/main/java/org/codice/ddf/security/claims/attributequery/common/AttributeQueryClaimsHandler.java

```diff
@@ -204,11 +178,9 @@ protected ProcessedClaimCollection createClaims(
       for (String claimType : supportedClaims) {
         if (claimType.equalsIgnoreCase(attribute.getName())) {
           String claimValue = attribute.getDOM().getTextContent();
-          if (attributeMap.containsKey(claimValue)) {
-            claimsCollection.add(createSingleValuedClaim(claimType, attributeMap.get(claimValue)));
-          } else {
-            claimsCollection.add(createSingleValuedClaim(claimType, claimValue));
-          }
+          claimsCollection.add(
+              createSingleValuedClaim(
+                  claimType, attributeMap.getOrDefault(claimValue, claimValue)));
```

Comment:
❓ Did you mean to use `claimValue` as both the key and the default?

Reply:
Yes, it lets you map known values over to whatever the system expects them to be.